src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileMetadataRepository.scala

```diff
@@ -1,16 +1,30 @@
 package uk.gov.nationalarchives.tdr.api.db.repository
 
+import java.util.UUID
+
 import slick.jdbc.PostgresProfile.api._
-import uk.gov.nationalarchives.Tables.{Filemetadata, FilemetadataRow}
+import uk.gov.nationalarchives.Tables.{File, Filemetadata, FilemetadataRow, Fileproperty}
 
 import scala.concurrent.Future
 
 class FileMetadataRepository(db: Database) {
   private val insertQuery = Filemetadata returning Filemetadata.map(_.fileid) into
     ((fileMetadata, fileid) => fileMetadata.copy(fileid = fileid))
-
+  private val propertyId = UUID.fromString("7a1b272c-e2f7-4b8f-8291-5e9dc312edb7")
```

Comment:
Can this be deleted now you're filtering on the property name?

Reply:
Oh yes, it can. Thanks!